docs/release-notes.md

```diff
@@ -21,6 +21,7 @@ Minor changes:
 - Add release notes to documentation
 - iso: Detect incomplete ISO files
 - Warn if console kargs could have used `--console`/`--dest-console` instead
+- install: bail when `--save-partindex` is specified on a MBR disk
```

Comment:
The release notes and commit message both refer to `--save-partindex` specifically, but the code also activates if `--save-partlabel` is specified.  We should decide which approach we want to take.